lightning/src/chain/channelmonitor.rs

```diff
@@ -184,6 +195,11 @@ impl_writeable_tlv_based_enum_upgradable!(MonitorEvent,
 		(2, monitor_update_id, required),
 		(4, channel_id, required),
 	},
+	(5, HolderForceClosedWithInfo) => {
+		(0, reason, upgradable_required),
+		(2, outpoint, required),
+		(4, channel_id, required),
+	},
```

Comment:
Move below `HolderForceClosed`?

Reply:
I can't get the marco to work if I move it. I think the ones with nested fields need to be first